src/views/Dashboard/Customer/Home/Emr/index.vue

```diff
@@ -236,22 +235,10 @@ export default {
           return filtered
         }, [])
 
-        if (listEMR.length > 0) {
-          listEMR.reverse() // TODO: BAD way, Need reverse from backend
-          for (let i = 0; i < listEMR.length; i++) {
-            const emrDetail = await this.metamaskDispatchAction(queryElectronicMedicalRecordById,
-              this.api,
-              listEMR[i]
-            )
-
-            if (emrDetail !== null) {
-              this.prepareEMRData(emrDetail)
-            }
-          }
-
-          this.emrDocuments.sort(
-            (a, b) => parseInt(b.timestamp) - parseInt(a.timestamp)
-          )
+        listEMR.reverse() // TODO: Remove this, it should be sorted from the backend side
```

Comment:
Why is there a `To Do` here? If you get the data directly from the blockchain I don't think it can be sorted.

Reply:
Noted, there I just thought that it could be sorted from BE. Ok thanks for the input I will fix it asap 👍🏼